### DISCUSSION TITLE
Profile page information are not showing properly

### DISCUSSION THREAD

**Discussion by muj-i** [community/community#63986] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

![85E6331B-54DE-4C04-B42B-B6F636B68606](https://github.com/community/community/assets/99077742/658d8c5c-0856-49ed-aaeb-ce368e70115c)

**Comment by advayc** [comment]
Try deleting and redownloading the app on your phone or check for any updates on app store.

**Reply by muj-i** [reply]
Thanks for your reply. I have already done this kind of thing. It still has problems.
